media/css/protocol/components/_download-button.scss

```diff
@@ -91,7 +90,6 @@ ul.download-list {
 .linux .download-button .os_linux,
 .linux.x86.x64 .download-button .os_linux64,
 .windows .download-button .os_win,
-.windows.arm .download-button .os_win64-aarch64,
```

Comment:
I am confused about everything we'd like to remove in this PR. Are we removing `os_win64-aarch64` detection as well? Or only instances of `windows.arm` AND `os_win64-aarch64`?
I ask because I noted that there are instances of `assert pq(list[2]).attr('class') == 'os_win64-aarch64'` in our download button tests in `test_helpers.py`

Reply:
We're removing UA detection for Arm64 on Windows. We're not removing Arm64 Windows builds, which we still produce and are available via firefox/all/